### PR TITLE
Documentation cleanup

### PR DESCRIPTION
**Pull request by malept**
* Reworded a bunch of things
* Removed the duplicate CLI usage section, added a link to usage.txt
* More consistent formatting

I'll rebase before this gets merged.

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -80,89 +47,102 @@ packager(opts, function done (err, appPath) { })
...
-`icon` - *String*
+  Not required if `all` is used.
 
-  Currently you must look for conversion tools in order to supply an icon in the
-  format required by the platform:
+`version` - *String*
 
-  - OS X: `.icns`
-  - Windows: `.ico`
+  Electron version (without the 'v'). See [Electron releases](https://github.com/atom/electron/releases) for valid versions.
```
Comment (maxogden) [inline readme.md]: maybe we should have an example here just to hammer it home to people

File: readme.md [hunk 2]
```diff
@@ -8,69 +8,36 @@ Package your electron app in OS executables (.app, .exe, etc) via JS or CLI. Sup
...
-                   - ProductVersion
-                   - ProductName
-                   - InternalName
+### From the Command Line
 
+Running electron-packager from the command line has this basic form:
 
 ```
+electron-packager <sourcedir> <appname> --platform=<platform> --arch=<arch> --version=<Electron version> [optional flags...]
+```
```
Comment (maxogden) [inline readme.md]: maybe have a real world example in addition to the template form, so its somewhat copy-pasteable (will also hopefully illustrate to people correct usage with `--version`)

File: readme.md (hunk 1 above)
Comment (malept) [inline readme.md]: :+1:

File: readme.md (hunk 2 above)
Comment (malept) [inline readme.md]: :+1: I think what I want to do is a sub-section with an example command, which would also explain exactly what would happen. Output directory name, abbreviated tree, etc. Make it painfully obvious.

**Comment by malept** [comment]
Added detailed CLI example and an example Electron version number.

**Comment by malept** [comment]
I'll rebase and merge this tomorrow unless there are any objections.

**Pull request merged**